shardingsphere-infra/shardingsphere-infra-common/src/test/resources/META-INF/services/org.apache.shardingsphere.infra.metadata.model.physical.jdbc.handler.JDBCSchemaHandler

```diff
@@ -0,0 +1,18 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+org.apache.shardingsphere.infra.metadata.model.physical.jdbc.handler.impl.OracleJDBCSchemaHandler
```

Comment:
It is unnecessary to add SPI duplicately.

Reply:
I got it，In MetaDataConnectionAdapter.java there is [ShardingSphereServiceLoader.register(JDBCSchemaHandler.class);] SPI load. how to Add test case for MetaDataConnectionAdapter.getSchema with SPI, please give advice，thanks @terrymanu 。

Reply:
The class OracleJDBCSchemaHandler has already loaded by ShardingSphereServiceLoader in prod code, so maybe we do not need to config it in test code again